submit_and_compare/submit_and_compare.py

```diff
@@ -147,8 +196,20 @@ def student_submit(self, submissions, suffix=''):
         '''
         Save student answer
         '''
+        if self.max_attempts > 0 and self.count_attempts >= self.max_attempts:
+            raise StandardError(
```

Comment:
the reason raising an error makes sense to me is because xblocks handle errors gracefully by default. We would have to add additional logic for what to return when the request is bad. Instead, throwing an results in the behavior shown in the screenshot below.

![screen shot 2016-05-17 at 8 08 49 pm](https://cloud.githubusercontent.com/assets/1752973/15346304/6d58b2ea-1c6b-11e6-87b4-fe8000a0438f.png)